README.rst

```diff
@@ -0,0 +1,67 @@
+==============================
+REANA Environment VOMSproxy
+==============================
+
+About
+=====
+
+``reana-auth-vomsproxy`` provides a container image with encapsulated runtime
+execution environment for creating a VOMS proxy certificate. The container image
+includes no additional logic or libraries, just the bare minimum to support the VOMS client. 
+The image is configured to support authentication for the four experiments at CERN; ALICE, ATLAS, CMS and LHCb.
+
+``reana-auth-vomsproxy`` was developed for use in the `REANA <http://reana.readthedocs.io/>`_ reusable research data analysis platform.
+
+Usage
+=====
+
+You can use ``reana-auth-vomsproxy`` as a base image, however it was built
+to be used as a sidecar container with the single purpose of refreshing the proxy certificate. 
+Once obtained, the certificate is shared with the main container using common namespaces. 
+
+Some concrete usage examples will be provided later on.
+
+Development
+===========
+
+You can build the ``reana-auth-vomsproxy`` image by passing the environment variables ``DATE`` in
+the format "YYYY-MM-DD" and ``VERSION`` as either a semantic versioning and/or the git SHA.
+
+Running the container and successfully obtaining a VOMS proxy certificate requires additional information and inputs:
+
+- Grid user certificate: ``${HOME}/.globus/usercert.pem``
+- Grid user key: ``${HOME}/.globus/userkey.pem``
+- Grid user password
+- Virtual organisation membership (e.g. for CMS: ``cms``)
+
+If you would like to try it locally, you can run:
+
+.. code-block:: console
+
+   $ docker run -i -t --rm -v $HOME/foo:/root/.globus/ reana-auth-vomsproxy /bin/bash
+
+
+Your local directory ``/foo`` should contain your ``usertcert.pem`` and ``userkey.pem`` files. By default the VOMS client
+checks the directory ``/$HOME/.globus`` for the files needed. In this image that path results in ``/root/.globus/``.
+
+A proxy can then be obtained by specifying the Virtual Organization, for example via:
+
+.. code-block:: console
+
+  $ voms-proxy-init --voms cms
+
+
+More information
+================
+
+For more information about `REANA <http://reanahub.io/>`_ reusable research data
+analysis platform, please see `its documentation
+<http://reana.readthedocs.io/>`_.
+
+Contributors
+============
+
+The list of contributors in alphabetical order:
+
+- `Adelina Lintuluoto` <adelina.eleonora.lintuluoto@cern.ch>
+- `Clemens Lange <https://orcid.org/0000-0002-3632-3157>`_ <clemens.lange@cern.ch>
```

Comment:
To make this process less tedious, as we expect the `reana-env`'s to be created "often", should we benefit from [repository templates](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-template-repository) ? CC @tiborsimko

Reply:
Also, there are substantial differences in terms of CI. VOMS proxy runs on GitHub actions and `reana-env-jupyter` on Travis, should Kerberos run on Travis or GitHub?

Reply:
> To make this process less tedious, as we expect the reana-env's to be created "often", should we benefit from repository templates ?

Actually this is `reana-auth-...` which we don't assume to be many... For HEP runtime containers, there could be more. We can definitely add some templates, or even cookiecutter with live questions to the person creating the repository, later.